agent/commands/setup.go

```diff
@@ -46,9 +46,13 @@ func Setup() {
 
 	l := logrus.WithField("component", "setup")
 	cfg, configFilepath, err := config.Get(l)
+
 	var e *config.ConfigFileDoesNotExistError
 	if err != nil && !errors.As(err, &e) {
-		fmt.Printf("Failed to load configuration: %s.\n", err) //nolint:forbidigo
+		err = nil
+	}
+	if err != nil {
+		fmt.Printf("Failed to load configuration: %s.\n", err)
```

Comment:
now it works incorrectly, because `!errors.As(err, &e)` checks if error is not `*config.ConfigFileDoesNotExistError`.
Your first fix was correct.

Reply:
yep, I fixed like @artemgavrilov suggested